utils/prometheus-node-exporter-lua/files/usr/lib/lua/prometheus-collectors/hostapd_ubus_stations.lua

```diff
@@ -1,18 +1,18 @@
 local ubus = require "ubus"
 local bit = require "bit"
 
-local function get_wifi_interfaces() -- based on hostapd_stations.lua
-  local u = ubus.connect()
-  local status = u:call("network.wireless", "status", {})
-  local interfaces = {}
+local function get_wifi_interfaces()
+  local handle = io.popen("ubus list hostapd.*")
```

Comment:
popen is "extremely" costly, please do it in lua
https://openwrt.org/docs/techref/ubus#lua_module_for_ubus
(`conn:objects()` part)

Reply:
btw you can't pass the filter (hostapd.*) yet, just sent a patch: http://lists.openwrt.org/pipermail/openwrt-devel/2022-May/038557.html

Reply:
You mean `conn:objects('hostapd.*')`?

Reply:
> You mean `conn:objects('hostapd.*')`?

yes, this will be supported when my patch land, but you can use `conn:objects('hostapd.*')` right now and filter using gmatch, so when the patch land it'll be a bit more efficient, and we can backport without waiting for the ubus patch to be backported

Reply:
Can you give me the gmatch code?